itest/assets_test.go

```diff
@@ -154,12 +157,17 @@ func mintAssetsConfirmBatch(t *harnessTest, tapd *tapdHarness,
 	require.NoError(t.t, err)
 	require.NotEmpty(t.t, batchResp.BatchKey)
 
+	mempoolWaitTimeout := defaultWaitTimeout
```

Comment:
This should probably also be set on the context timeout above. But the real thing I ran into when running this locally is this: https://github.com/lightninglabs/taproot-assets/blob/0a58cdbf873f4634f05aa6f40fa75f888f0f5195/tapgarden/caretaker.go#L805
This will cancel the context after 1 minute. So we should probably make that `WithCtxQuitNoTimeout`...